.travis.yml

```diff
@@ -39,49 +39,55 @@ matrix:
       env: VERSION=5.0  BUILD=RelWithDebInfo
     # 5
     - compiler: clang
-      env: VERSION=5.0  BUILD=RelWithDebInfo  INVERTED=ON
+      env: VERSION=6.0  BUILD=RelWithDebInfo
     # 6
+    - compiler: clang
+      env: VERSION=6.0  BUILD=RelWithDebInfo  INVERTED=ON
+    # 7
     - compiler: gcc
       env: VERSION=7    BUILD=RelWithDebInfo  INVERTED=ON
-    # 7
+    # 8
     - compiler: gcc
       env: VERSION=7    BUILD=RelWithDebInfo
-    # 8
+    # 9
     - compiler: gcc
       env: VERSION=6    BUILD=RelWithDebInfo
-    # 9
+    # 10
     - compiler: gcc
       env: VERSION=5    BUILD=RelWithDebInfo
-    # 10
+    # 11
     - compiler: gcc
       env: VERSION=4.8  BUILD=RelWithDebInfo
-    # 11
+    # 12
     - compiler: clang
       os: osx
       osx_image: xcode9.2
       env:              BUILD=Debug           MAINTAINER_MODE=OFF
-    # 12
-    - compiler: clang
-      env: VERSION=4.0  BUILD=Debug
     # 13
     - compiler: clang
-      env: VERSION=5.0  BUILD=Debug
+      env: VERSION=4.0  BUILD=Debug
     # 14
     - compiler: clang
-      env: VERSION=5.0  BUILD=Debug           INVERTED=ON
+      env: VERSION=5.0  BUILD=Debug
     # 15
+    - compiler: clang
+      env: VERSION=6.0  BUILD=Debug
+    # 16
+    - compiler: clang
+      env: VERSION=6.0  BUILD=Debug           INVERTED=ON
+    # 17
     - compiler: gcc
       env: VERSION=7    BUILD=Debug           INVERTED=ON
-    # 16
+    # 18
     - compiler: gcc
       env: VERSION=7    BUILD=Debug
-    # 17
+    # 19
     - compiler: gcc
       env: VERSION=6    BUILD=Debug
-    # 18
+    # 20
     - compiler: gcc
       env: VERSION=5    BUILD=Debug
-    # 19
+    # 21
```

Comment:
Not directly related to your commit but could you please also update `JOB_NUMBERS` to `1 .. 21` - for some reason it's `1 .. 24` now